values-schema.yaml

```diff
@@ -482,6 +572,10 @@ definitions:
         type: string
       secrets:
         $ref: '#/definitions/secrets'
+      externalSecrets:
```

Comment:
like I mentioned above, I thought we would replace it entirely for the new handling...

Reply:
we agreed on using externalSecrets for now

Reply:
Can you make sure the secrets solution of yours replaces the current solution entirely? We don't expect them side by side. So the final commit in this PR should reflect that.